app/page.tsx

```diff
@@ -22,34 +25,60 @@ export default function Home() {
     getRules();
   }, []);
 
-  const mappedRules = rules.map(({ _id, title, goRulesJSONFilename }) => {
+  const mappedRules = rules.map(({ _id, title, goRulesJSONFilename, isPublished }) => {
+    const ruleLink = `/rule/${_id}`;
+    const draftLink = `${ruleLink}?version=draft`;
     return {
       key: _id,
       titleLink: (
         <b>
-          <a href={`/rule/${_id}`}>{title || goRulesJSONFilename}</a>
+          <a href={isPublished ? ruleLink : draftLink} className={styles.mainLink}>
+            {title || goRulesJSONFilename}
+          </a>
         </b>
       ),
-      downloadRule: (
-        <a href={`/api/documents?ruleFileName=${encodeURIComponent(goRulesJSONFilename)}`}>Download JSON</a>
+      draft: (
```

Comment:
Would it feel counterintuitive to have a 'published' button as well? It feels fairly clear that you can click on the name of the rule to go to the published version, but it doesn't outright say that, and the other versions get their own dedicated button - so I wonder if there might be some user confusion.

Reply:
Yeah, I thought about that. Didn't want to overcrowd it, but I think I'll add it. Realize I'm currently showing all the buttons even if they don't apply for that rule, so I'm pushing an update that changes that and reformats it a bit too.